INSTALL.md

```diff
@@ -118,6 +113,17 @@ they are using your local Elasticsearch)
 rake es:delete[_all]
 ```
 
+The data directory can optionally include a file called `data.yaml` (see [the sample one](sample-data/data.yaml) for its schema) that references one or more `.csv` files and specifies data types,
+field name mapping, and other.
```

Comment:
I copied that from somewhere... I think we need a lot more on the data.yml format, but that is a subject for another PR!